### PR TITLE
DOCS-942: Add documentation for GetProperties() to base component

### PR DESCRIPTION
**Pull request by sguequierre**
Implements the third segment of the changes outlined here: [Docs Changes for GPS Navigation](https://docs.google.com/document/d/1ReL7RU0FoaAe8gyPbMgMovHpgdx3Jy4c3R0LFCVWkKI/edit) (section below "Changes to https://docs.viam.com/components/base/")

* Add the GetProperties() method with small code samples, add info for navigation service usage

### CODE REVIEW

File: docs/components/base/_index.md
Comment (npentrel) [inline docs/components/base/_index.md:519]: No I think this is the actual code you'd be reading

**Review by npentrel — APPROVED** [review]
LGTM once Tess approves. And % one modification to either the link or the link text in line 454

File: docs/components/base/_index.md
Comment (sguequierre) [inline docs/components/base/_index.md:519]:
```suggestion
For more information, see the [Go SDK Code](https://github.com/viamrobotics/rdk/blob/main/resource/resource.go).
```

File: docs/components/base/_index.md
Comment (tessavitabile) [inline docs/components/base/_index.md:417]: GetProperties is only supported in Go currently.

Comment (sguequierre) [reply docs/components/base/_index.md:417]: https://python.viam.dev/autoapi/viam/components/base/client/index.html#viam.components.base.client.BaseClient.get_properties

File: docs/components/base/_index.md
Comment (sguequierre) [inline docs/components/base/_index.md:410]:
```suggestion
Get the width and turning radius of the {{< glossary_tooltip term_id="model" text="model" >}} of base in meters.
```

File: docs/components/base/_index.md
Comment (sguequierre) [inline docs/components/base/_index.md:414]:
```suggestion
```

File: docs/components/base/_index.md
Comment (sguequierre) [inline docs/components/base/_index.md:411]:
```suggestion
```

File: docs/components/base/_index.md
Comment (tessavitabile) [reply docs/components/base/_index.md:413]: Oh that's surprising. https://viam.atlassian.net/browse/RSDK-3321 is still in triage, but it does look like this was implemented. @cheukt can you clarify if GetProperties on a base is supported in python?

Comment (cheukt) [reply docs/components/base/_index.md:413]: base.get_properties is supported currently, it was done as part of a different ticket.

File: docs/components/base/_index.md
Comment (randhid) [inline docs/components/base/_index.md:83]: do you want to add the units here as well?

**Review by randhid — APPROVED** [review]
LGTM! Yes, can confirm get_properties was added to python, but not other sdks.

File: docs/components/base/_index.md
Comment (sguequierre) [reply docs/components/base/_index.md:83]: Yeah I was going back and forth I can do that

Comment (sguequierre) [inline docs/components/base/_index.md:83]:
```suggestion
| [GetProperties](#getproperties) | Get the width and turning radius of the base in meters. |
```

Comment (sguequierre) [reply docs/components/base/_index.md:83]: done!

File: docs/program/apis/_index.md
Comment (sguequierre) [inline docs/program/apis/_index.md:224]:
```suggestion
| [GetProperties](/components/base/#getproperties) | Get the width and turning radius of the base in meters. |
```

File: docs/components/base/_index.md
Comment (tessavitabile) [reply docs/components/base/_index.md:413]: Thank you!

**Review by tessavitabile — APPROVED** [review]

**Pull request merged**
